### DISCUSSION TITLE
About github interface

### DISCUSSION THREAD

**Discussion by SIS1wang** [community/community#126928] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

Hi,I'm a new user of github.I'm having some problems with github.Why dose my github interface go blank when I set a github theme?My attempts to revert to no theme didn't help.I can't access my github at all through my github URL,and no one else can either.I really hope you can help me solve this problem.I would love to use my github right now.

**Comment by davevad93** [comment]
Hi @SIS1wang ,

That's a strange behavior, it could be something related to cookies or browser cache, try the following:

1. Use another browser.
2. Use the incognito mode of your browser.

About the other issue, I have no problem at all accessing your profile through the URL https://github.com/SIS1wang

**Reply by SIS1wang** [reply]
First of all,thank you very much for your reply.I am curious why I open through this website ( https://SIS1wang.github.io) is blankWhether I open it on my phone or Google Chrome,the screen is blank.But it's OK for me to go to this page before I change the theme.Of course ,I applied for github because my upperclassman told me to access my homepage through the URL I just wrote.So I don't know how solve this problem.

**Reply by davevad93** [reply, marked as answer]
Hey @SIS1wang you're welcome. Now I understand that you are talking about setting up you personal website via [SIS1wang.github.io](https://github.com/SIS1wang/SIS1wang.github.io) repo. The problem is that your `index.html` is empty, that's why the website is blank, you have to put your main `html` code there in order for the website to render correctly. Take a look a this from the docs on how to properly set up a web using GitHub pages:

1. [About GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)
2. [Creating a GitHub Pages site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)

**Reply by SIS1wang** [reply]
Oh my God,thank you very much,now I finally konw why my website is blank,I will solve this problem now,thank you again for your reply.

**Reply by davevad93** [reply]
You're welcome @SIS1wang , have a nice day!

**Reply by SIS1wang** [reply]
Have a nice day,too!Thank you~
